src/Sulu/Component/Rest/Tests/Unit/MockInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\Tests\Unit;
+
+interface MockInterface
```

Comment:
This class was used in the RestHelperTest. Phpunit 10 will not allow to Mock classes that don't exist so, just created this interface to use it in mocking.